packages/schematics/src/collection/node-application/files/nestjs/src/app.controller.spec.ts__tmpl__

```diff
@@ -0,0 +1,22 @@
+import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
Not pertaining just to this file, but for these `nest` modules, controllers, and services: Can they go into the `app` directory similar to the `Angular` directory structure?

Reply:
can do